spec/unit/generator/xcconfig/pod_xcconfig_spec.rb

```diff
@@ -86,7 +86,7 @@ module XCConfig
           it 'sets the PODS_TARGET_SRCROOT build variable for local pod' do
             @pod_target.sandbox.store_local_path(@pod_target.pod_name, @spec.defined_in_file)
             @xcconfig = @generator.generate
-            @xcconfig.to_hash['PODS_TARGET_SRCROOT'].should == '${PODS_ROOT}/../../spec/fixtures/banana-lib/BananaLib.podspec'
+            @xcconfig.to_hash['PODS_TARGET_SRCROOT'].should == '${PODS_ROOT}/../../spec/fixtures/banana-lib'
```

Comment:
This seems like the original implementation was incorrect, as I don't think the full podspec path should be the value of `PODS_TARGET_SRCROOT`

Reply:
I think this was just a bad test, the actual generated value is correct, right?

Reply:
Not sure, how was the testing passing before then?